### PR TITLE
feat: disable join modal

### PR DESCRIPTION
**Pull request by frattezi**
# Description

Quick PR disabling the "join PodCodar" feature on the website, as decided by the board.

## Changes

- Remove `CallToActionModal` usage and disable Join button at the `NavBar`.
- Remove call-to-action button `Join Us`
- Add informative tooltip.

### CODE REVIEW

File: packages/locale/en.yml
Comment (marco-souza) [inline packages/locale/en.yml:3]: I think we have type definitions for this file. Let's check it before merging

**Review by marco-souza — APPROVED** [review]
👍

**Comment by rafaellvs** [comment]
should we also remove the "Join us" button at `<CallToActionSection />`? Its not doing anything in the preview

**Comment by frattezi** [comment]
> should we also remove the "Join us" button at `<CallToActionSection />`? Its not doing anything in the preview

You're right skinny bud, I will remove the button for now!

File: packages/locale/en.yml
Comment (frattezi) [reply packages/locale/en.yml:3]: Marco I don't see type definitions for this 🤔 Are you sure?

We have a d.ts module declaration for locales:

```ts
declare module '@packages/locale/pt.yml';
declare module '@packages/locale/en.yml';
```

Am I missing something else I should do? If so can you please point it out? Thank you 🙇🏼

Comment (frattezi) [reply packages/locale/en.yml:3]: Double checked, as far as I can see we're ok here, let me KWYT, ty

**Review by vitorschelb — APPROVED** [review]
Thats great Frattezi, thank you so much!

File: packages/locale/en.yml
Comment (marco-souza) [reply packages/locale/en.yml:3]: Hmmm sounds good, I just remembered that this typing is Garbage and it's my fault 😅, sorry about that

Comment (frattezi) [reply packages/locale/en.yml:3]: No worries, it's usage is simple, not suffering for missing types there, LGTM

**Pull request merged**
